bacon/model/StorageCouchBaseDB.swift

```diff
@@ -21,11 +21,15 @@ class StorageCouchBaseDB {
 
     // MARK: - Properties
     private var transactionDatabase: Database
+    // Dictionary to provide a mapping from instantiated `Transaction` objects
+    // to their unique id in the databse.
+    private var transactionMapping: [Transaction: String]
```

Comment:
Maybe call this transactionIdMap to make it more descriptive what is being mapped to what